app/src/main/res/layout/activity_course_detail.xml

```diff
@@ -1,52 +1,64 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/relativeLayout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
     tools:context=".ui.home.ExploreFragment">
 
-    <LinearLayout android:layout_width="match_parent"
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Please avoid nesting in constraint layout. Nesting decreases app's performance. Also please change the layout IDs accordingly.

Reply:
> Please avoid nesting in constraint layout. Nesting decreases app's performance. Also please change the layout IDs accordingly.
 so I should remove the nested constraintlayout?

Reply:
Yes. You need to remove that. But the app design should look same and everything should work.